docs/src/modules/components/MarkdownElement.js

```diff
@@ -458,8 +466,8 @@ const Root = styled('div')(
       height: 26,
       cursor: 'pointer',
       position: 'absolute',
-      top: theme.spacing(1.5),
-      right: 12,
+      top: 12,
+      right: 16,
```

Comment:
For the spacing to feel balanced? With the latest change, doesn't feel needed

```suggestion
      right: 12,
```